install/kubernetes/helm/istio/values.yaml

```diff
@@ -454,3 +454,12 @@ global:
   # This field is set to false by default, so 'helm template ...'
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
+
```

Comment:
can you delete these 3 carriage returns.

Reply:
Done